.github/CODEOWNERS

```diff
@@ -0,0 +1,43 @@
+# Remember: order matters. Subsequent rules will override prior rules.
+
+*assert* @nodejs/testing
+*async_hook* @nodejs/async_hooks @nodejs/diagnostics
+*async_wrap* @nodejs/async_hooks
+*buffer* @nodejs/buffer
+*child_process* @nodejs/child_process
+*cluster* @nodejs/cluster
+*crypto* @nodejs/crypto
+*eslint* @nodejs/linting
+*fs* @nodejs/fs
+*tls* @nodejs/tls
+*dgram* @nodejs/dgram
+*domain* @nodejs/domains
+*fs* @nodejs/fs
+*http* @nodejs/http @nodejs/http2
+*inspector* @nodejs/V8-inspector
+*module* @nodejs/modules
+*node_api* @nodejs/n-api
+*repl* @nodejs/repl
+*timer* @nodejs/timers
+*trace_events* @nodejs/trace-events
+*url* @nodejs/url
+*util* @nodejs/util
+*vm* @nodejs/vm
+*zlib* @nodejs/zlib
+
+/lib/internal/modules/esm/ @nodejs/modules
```

Comment:
Is it covered by `*module* @nodejs/modules`?

Reply:
But will not it be pinged as `*module*` glob above matches`/lib/internal/modules/cjs/`?

Reply:
Yeah, that will need to be refined. We don't have a team for the cjs module system. That one might need to be nodejs/tsc

Reply:
(Then [COLLABORATOR_GUIDE.md#who-to-cc-in-the-issue-tracker](https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md#who-to-cc-in-the-issue-tracker) also needs to be refined as for `@nodejs/modules` cc)